lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -600,7 +597,7 @@ def test_manage_student_enrollment_limit(self):
         # enroll the first student
         data = {
             'student-action': 'add',
-            'student-id': u','.join([students[0].email, ]),
+            'student-id': u''.join([students[0].email, ]),
```

Comment:
Can you change this to just `students[0].email`? `student-id` is not meant to be a list of anything